### PR TITLE
[2021.1] SDET backports

### PR DESCRIPTION
**Pull request by theopnv**
### Purpose of this PR

Backported 
- #4240: Refactor templates pipelines 
- #4442: Yamato - Clean old pipelines 

---
### Testing status
- package and preview publish jobs were removed from the yamato listing
- Publish HDRP template job (dry run): https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/2021.1%252F2021-05-05%252Fsdet%252Fmisc-backports/.yamato%252F_templates.yml%2523publish_hdrp_template_2021.1_dry_run/6665196/job
- Publish URP template job (dry run): https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/2021.1%252F2021-05-05%252Fsdet%252Fmisc-backports/.yamato%252F_templates.yml%2523publish_universal_template_2021.1_dry_run/6679216/job

### CODE REVIEW

**Comment by JanBorupJakobsen** [comment]
Dry run for Publish URP templates failed ?

**Comment by theo-pnv** [comment]
Sorry, I forgot to update the link in the PR description.
The last rerun is working https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/2021.1%252F2021-05-05%252Fsdet%252Fmisc-backports/.yamato%252F_templates.yml%2523publish_universal_template_2021.1_dry_run/6679216/job

File: com.unity.template-universal/Packages/com.unity.template.universal/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ All notable changes to this project template will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [11.0.0] - 2020-10-21
```
Comment (theo-pnv) [inline com.unity.template-universal/Packages/com.unity.template.universal/CHANGELOG.md:7]: Note that I had to add this entry to fix the URP job.
I think it was forgotten by mistake when we bumped all the changelogs.
With that fix, the publish job works.

File: .yamato/_projectcontext.yml [hunk 2]
```diff
@@ -378,8 +378,6 @@ all_package_ci_project_2021.1:
     dependencies:
       -  .yamato/_projectcontext.yml#test_all_project_Win_2021.1
       -  .yamato/_projectcontext.yml#test_all_project_OSX_2021.1
-    triggers:
-        expression: pull_request.target eq "2021.1/staging" AND NOT pull_request.draft
```
[the remark this answers is not in the document]
Comment (theo-pnv) [reply .yamato/_projectcontext.yml]: Thanks, fixed

**Pull request merged**